blogs/2017/10/24/theicon.md

```diff
@@ -0,0 +1,145 @@
+
+For complete transparency, I want to share some history as we think it helps to understand how we arrived at orange in the first place.
+
+## Welcome to the family
+Back in [March](https://blogs.msdn.microsoft.com/visualstudio/2017/03/08/iterations-on-infinity/) we (the broader Visual Studio team) set out to create a "family" feel across all of the developer tools we produce. If you read that blog post again, you can see that we had to work within a set of "challenging" corporate branding guidelines (e.g. a single color).
+
+It is easy to blame the guidelines, but at the same time, having a set of products that are easily and broadly recognizable is a valuable asset. Instead, we took on the challenge of working within the constraints while also working across the company to evolve the guidelines and address the problems we faced.
+
+When we saw the negative feedback from the initial set of icons, we went heads down on the Code icon. We wanted to produce an icon that both the broad community and we would find instantly, visually, and emotionally appealing.
+
+Here are some of the early designs:
+
+![Early VS Code icon designs](2017_10_24_earlydesigns.png)
+
+None of these really stood out for us, except the one in the upper right corner.
+
+![Early VS Code icon designs](2017_10_24_editorlinesicon.png)
+
+This was rejected because it didn't contain any folds, which is a component of the family design. Back to the drawing board.
+
+We iterated on the infinity logo for some time, for example:
+
+![Variations on the infinity design](2017_10_24_variationsoninfinity.png)
+
+Including one that will hurt your eyes:
+
+![A variation that hurts the eyes](2017_10_24_hurtsmyeyes.png)
+
+Until we landed on the current "fish", which tested well with admittedly, a small number of developers.
+
+## Rotation
+
+We have a lot more design freedom on the Mac (and Linux), which is why we have a different icon there. While we wanted to keep the same icon shape, we were not limited to a single color and we could resolve contrast issues by mounting the icon on a dark panel. The rotation actually adheres to the [Apple design guidelines](https://developer.apple.com/macos/human-interface-guidelines/icons-and-images/app-icon).
+
+That said, the feedback that the rotation is outdated and the mount makes the icon appear smaller than other icons in the Dock is completely fair.
+
+## Fatigue
+By mid-summer, icon fatigue rolled in. We had made so many iterations, we finally said let's go with the current shape, and we checked in the "pre-release" design for Insiders. A pre-release is denoted by the hashed box on the right and bottom:
+
+![Pre-release icons in orange](2017_10_24_orangeprerelease.png)
```

Comment:
I don't think the images are linked to the right folder

Reply:
The blogs directory structure means the pages don't render images in GitHub or local view. The blog post will be on PPE shortly. 
Pine has a pending commit to fix the blog image problem.